README.md

```diff
@@ -147,36 +145,4 @@ Pull requests are welcome 👍. For major changes, please open an issue first to
 
 [MIT](https://choosealicense.com/licenses/mit/)
 
-
 ------------------------------------
-
-## Contributors ✨
```

Comment:
This will remove add contributor bots bedge.

Reply:
I don't know why that happened